products/rds-mysql.md

```diff
@@ -0,0 +1,66 @@
+---
+title: RDS mysql
+category: db
+iconSlug: mysql
+permalink: /rds-mysql
+releasePolicyLink: >-
+  https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html
+releaseImage: https://docs.aws.amazon.com/assets/r/images/aws_logo_dark.png
+activeSupportColumn: Active Support
+releases:
+  - releaseCycle: "8.0.32"
```

Comment:
Although Amazon [document support for patch version](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html#MySQL.Concepts.VersionMgmt.Supported), that's not something that is usually done on endoflife.date because it is really hard to maintain.

Depending on what you need (do not hesitate to add details), maybe you should only document the major versions EOL (https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MySQL.Concepts.VersionMgmt.html#MySQL.Concepts.VersionMgmt.ReleaseCalendar) ?

@captn3m0 any thought ?

Reply:
I switched this to only the major version now. I added data for the latest column. I also have a consideration I wrote out in the [comment here](https://github.com/endoflife-date/endoflife.date/pull/2626#issuecomment-1454886822)